app/data_model/list_store.py

```diff
@@ -39,6 +40,14 @@ def __getitem__(self, list_item_index):
     def __len__(self):
         return len(self.items)
 
+    @cached_property
+    def non_primary_people(self):
+        non_primary = []
+        for item in self.items:
+            if item != self.primary_person:
+                non_primary.append(item)
+        return non_primary
```

Comment:
Could be simplified using a list comprehension.
```suggestion
        return [item for item in self.items if item != self.primary_person]
```